geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/pubsub/PubSubIntegrationTest.java

```diff
@@ -20,6 +20,11 @@
 import org.apache.geode.redis.GeodeRedisServerRule;
 
 public class PubSubIntegrationTest extends AbstractPubSubIntegrationTest {
+
+  static {
+    System.setProperty("io.netty.eventLoopThreads", "10");
+  }
+
```

Comment:
Why could you not use `RestoreSystemProperties` here? 

This test does not clean up itself after setting the property. What would the effect be if you don't clean up this property to the "default" value?

Reply:
Good call. Should be fixed now.